tests/integration/SdkTest_test.cpp

```diff
@@ -4684,6 +4684,61 @@ TEST_F(SdkTest, SdkGetBanners)
     ASSERT_TRUE(err == MegaError::API_OK || err == MegaError::API_ENOENT) << "Get banners failed (error: " << err << ")";
 }
 
+TEST_F(SdkTest, SdkBackupFolder)
+{
+    getAccountsForTest(1);
+    LOG_info << "___TEST BackupFolder___";
+
+    //
+    // make this work even before the remote API has support for My Backups folder
+
+    // create My Backups folder
+    fetchnodes(0, maxTimeout);
+    std::unique_ptr<MegaNode> myBkpsNode{ megaApi[0]->getNodeByPath("/My Backups") };
+    if (!myBkpsNode)
+    {
+        std::unique_ptr<MegaNode> rootnode{ megaApi[0]->getRootNode() };
+        const char* folderName = "My Backups";
+        ASSERT_NO_FATAL_FAILURE(createFolder(0, folderName, rootnode.get()));
+
+        fetchnodes(0, maxTimeout);
+
+        myBkpsNode.reset(megaApi[0]->getNodeByPath("/My Backups"));
+
+        // set My Backups handle attr
+        MegaHandle mh = myBkpsNode->getHandle();
+        int err = synchronousSetMyBackupsFolder(0, mh);
+        ASSERT_TRUE(err == MegaError::API_OK) << "Setting My Backup folder failed (error: " << err << ")";
+        err = synchronousGetMyBackupsFolder(0);
+        ASSERT_TRUE(err == MegaError::API_OK);
+
+        // read the attribute to make sure it was set
+        std::unique_ptr<MegaUser> user(megaApi[0]->getMyUser());
+        ASSERT_NO_FATAL_FAILURE(getUserAttribute(user.get(), MegaApi::USER_ATTR_MY_BACKUPS_FOLDER, maxTimeout, 0));
```

Comment:
There is a dedicated interface to `getMyBackupsFolder()` that should be used, instead of the generic one for any user attribute (which should work as well, but may not set the MegaRequest members in the same way than the specific one).
You've already called it in line 4712, so you just need to check the value is the intended one. No need to call `getUserAttribute()`.